core/src/main/java/org/eclipse/hono/util/ProtocolAdapterConstants.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 1.0 which is available at
+ * https://www.eclipse.org/legal/epl-v10.html
+ *
+ * SPDX-License-Identifier: EPL-1.0
+ */
+package org.eclipse.hono.util;
+
+/**
+ * Constants used by protocol adapters of Hono.
+ *
+ */
+public final class ProtocolAdapterConstants {
```

Comment:
can't we simply add these to `Constants`?

Reply:
Yes, you are right.Will move them.